x-pack/plugins/security_solution/public/management/pages/policy/view/policy_details.tsx

```diff
@@ -245,6 +276,33 @@ export const PolicyDetails = React.memo(() => {
         <MacEvents />
         <EuiSpacer size="l" />
         <LinuxEvents />
+
+        {showAdvancedPolicy && (
+          <EuiButtonEmpty
+            data-test-subj="hideAdvancedPolicyButton"
+            onClick={handleHideAdvancedPolicyClick}
+          >
+            <FormattedMessage
+              id="xpack.securitySolution.endpoint.policy.advanced.hide"
+              defaultMessage="Hide advanced settings"
+            />
+          </EuiButtonEmpty>
+        )}
+
+        {!showAdvancedPolicy && (
+          <EuiButtonEmpty
+            data-test-subj="showAdvancedPolicyButton"
+            onClick={handleShowAdvancedPolicyClick}
+          >
+            <FormattedMessage
+              id="xpack.securitySolution.endpoint.policy.advanced.show"
+              defaultMessage="Show advanced settings"
+            />
+          </EuiButtonEmpty>
+        )}
+
+        <EuiSpacer size="l" />
+        {showAdvancedPolicy && <AdvancedPolicyForms />}
```

Comment:
Similarly to Protections and Settings above, we should add a header for Advanced to match the mock.  However, it should be hidden under the fold. 

Here's where the title is for Protections, for instance.

https://github.com/elastic/kibana/blob/b37f2afa476c28647ab682828103de59a9600756/x-pack/plugins/security_solution/public/management/pages/policy/view/policy_details.tsx#L251

Reply:
f44601c417470f867e7aff2f38871e449f948725